lua/rip-substitute/popup-win.lua

```diff
@@ -297,7 +314,12 @@ function M.openSubstitutionPopup()
 	local m = config.keymaps
 	local keymapHint = #state.popupHistory == 0
 			and ("%s confirm  %s abort"):format(m.confirm, m.abort)
-		or ("%s/%s history  %s literal"):format(m.prevSubst, m.nextSubst, m.toggleFixedStrings)
+		or ("%s/%s history  %s literal %s case sensitive"):format(
+			m.prevSubst,
+			m.nextSubst,
+			m.toggleFixedStrings,
+			m.toggleCaseSensitive
```

Comment:
Adding it to the hints here increases `minWidth` quite a bit, resulting in an unnecessarily wide window. 

I'd suggest to just leave it this one out for now, and maybe implement some system for rotating between hints to display in another PR.